py/selenium/webdriver/remote/webelement.py

```diff
@@ -101,7 +101,7 @@ def clear(self) -> None:
         """Clears the text if it's a text entry element."""
         self._execute(Command.CLEAR_ELEMENT)
 
-    def get_property(self, name) -> Union[str, bool, WebElement, dict]:
+    def get_property(self, name) -> Optional[Union[str, int, bool, WebElement, dict, list]]:
```

Comment:
I'm not sure this can return `None`

Reply:
It return `None` when invalid property name is used. Please read the PR description for more info.

Reply:
my apologies, i was thinking about element finding rather than property fetching!